communities.json

```diff
@@ -771,6 +771,12 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "vatsim.svg",
+      "title": "VATSIM",
+      "inviteCode": "4TnAVHHkqB",
```

Comment:
For some reason this invite code isn't working for me so I can't find your server. Might be because of some settings on the invite code. Could you:

1. Double check the code doesn't have an expiration on it
2. Send me your server's ID (https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)

Reply:
1. The invite did not have an expiration set, but it did only grant temporary membership. If that is also included in the "having an expiration" statement then I'll generate a new one which grants permanent membership.
2. 703923823887515720